app/cells/folio/console/layout/sidebar/show.slim

```diff
@@ -21,7 +21,10 @@ aside#f-c-layout-sidebar.f-c-layout-sidebar data-controller="f-c-layout-sidebar"
 
             ul.f-c-layout-sidebar__ul
               - group[:links].each do |link|
-                li.f-c-layout-sidebar__li
-                  == link
+                - if link == :hr
+                  hr
```

Comment:
`hr` uvnitr `ul` neni validni

proc jsi nepouzil to razeni do groups, jak jsem psal?

Reply:
protoze z toho co jsem vycetl z kodu, tak groups jsou ty jednotlivy sites a ne mnoziny linku pro jednu group - tak tady by sel fix aby bylo hr misto li?

Reply:
tak tam prosim pridejme dalsi uroven vnoreni, kde se pres css udelaji linky